pkg/i18n/english.go

```diff
@@ -115,6 +116,7 @@ func englishSet() TranslationSet {
 		EditConfig:         "edit lazydocker config",
 		Cancel:             "cancel",
 		Remove:             "remove",
+		HideStopped:        "Hide/Show stopped container",
```

Comment:
I think the plural makes more sense here
```suggestion
		HideStopped:        "Hide/Show stopped containers",
```